packages/cssnano/src/__tests__/fixtures.js

```diff
@@ -76,7 +76,7 @@ test(
     h2, h1 {
         font-weight: 400;
     }`,
-    `h1,h2{color:red;font-weight:400}`
+    `h1,h2{font-weight:400;color:red}`
```

Comment:
Why it is happens?

Reply:
Because `font-weight` has a shorthand `font`.

Reply:
Clarified this behaviour with a test: `move properties that are not in `safe-order.json` to the start and sort remaining` in bfe1257. Properties that are not 'safe', in other words properties that have a shorthand/longhand are moved to the start.